geometry_msgs/PoseStamped inPose
---
geometry_msgs/PoseStamped outPose
